src/index.js

```diff
@@ -334,14 +334,22 @@ function parseBody(ramlBody, srMethod) {
   if (_.isUndefined(srMethod.parameters))
     srMethod.parameters = [];
 
-  srMethod.parameters.push({
+  //Fix for the body parameter - empty schema to be set to empty object
+  //Schema is mandatory for body parameter
+  var bodyRequest = {
     //FIXME: check if name is used;
     name: 'body',
     in: 'body',
     required: true,
     //TODO: copy example
     schema: convertSchema(_.get(ramlBody[jsonMIME], 'schema'))
-  });
+  };
+
+  if (!bodyRequest.schema) {
+    bodyRequest.schema = {type: "object"};
```

Comment:
I wouldn't know from the top of my head if that works. But if the output will be valid Swagger we're totally fine with it.